corehq/apps/fixtures/dbaccessors.py

```diff
@@ -54,6 +54,16 @@ def iter_fixture_items_for_data_type(domain, data_type_id):
         yield FixtureDataItem.wrap(row['doc'])
 
 
+def count_fixture_items(domain, data_type_id):
```

Comment:
Looks like this is only used in tests, not a big deal to be here, but just a note.